src/test/scala/sss/db/DbSpecSetup.scala

```diff
@@ -21,19 +23,42 @@ trait DbSpecSetupBase extends Matchers with BeforeAndAfter {
   val testPaged2 = "testPaged2"
   val testRowSerialize = "testRowSerialize"
 
-  before {
+  override def beforeEach(): Unit = {
     val dbUnderTest = Db(dbConfigName, DataSource(s"$dbConfigName.datasource"))
     fixture = TestFixture(dbUnderTest, dbUnderTest.table("test"))
+    super.beforeEach()
   }
 
-  after {
+  override def afterEach(): Unit = {
     val db = fixture.dbUnderTest
     import db.syncRunContext
-    import db.syncRunContext.executor
-    fixture.dbUnderTest.shutdown.runSyncAndGet
+    try super.afterEach()
+    finally fixture.dbUnderTest.shutdown.runSyncAndGet
   }
 
+  def persistIntVal(intVal: Int): FutureTx[Row] = fixture.table.persist(Map("strId" -> "strId", "createTime" -> new Date(), "intVal" -> intVal))
+  def persistIntVals(intVals: Seq[Int]): FutureTx[Seq[Row]] = FutureTx.sequence(intVals map persistIntVal)
 }
 
 trait DbSpecSetup extends AnyFlatSpec with DbSpecSetupBase
+
+trait DbSpecQuickSetupBase extends DbSpecSetupBase {
```

Comment:
Added that to reduce boilerplate when using DbSpecSetup. Haven't changed DbSpecSetup itself, to avoid making a lot of changes at once. I didn't have a good naming idea, thus  the "Quick" in the middle.

Reply:
I see a db shutdown after each .... I have a nagging memory of something going wrong with that... please try to run the testsuite repeatedly locally watching for errors. Strange request admittedly.